cloud/blockstore/tools/testing/chaos-monkey/main.go

```diff
@@ -105,35 +105,35 @@ func run(ctx context.Context, opts *options) error {
 
 	var diskInfo *DiskInfo
 	diskInfo, err = describeDisk(state, opts.DiskID)
-	fmt.Println("DiskInfo:")
+	log.Println("DiskInfo:")
```

Comment:
consider printing in one line and not split it into two lines ... different go routines can interleave other data into the logs between the two lines

same remark to other places where you
```go
log.Println("title:")
...
log.Println(data)
```